docs/contracts/libraries.rst

```diff
@@ -38,6 +38,19 @@ To realize this in the EVM, code of internal library functions
 and all functions called from therein will at compile time be included in the calling
 contract, and a regular ``JUMP`` call will be used instead of a ``DELEGATECALL``.
 
+.. note::
+    Public library functions, unlike public functions inherited from contracts, cannot be
```

Comment:
I find this rather confusing. Maybe just mention that the above analogy fails in so far as public library functions cause an external (delegate)call when called like `L.f()`. In contrast, `A.f()` is an internal call when `A` is a base contract of the current contract.

Reply:
ok. I have rewritten it according to your suggestion.